broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -212,6 +507,32 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(ServiceBindingNotFound, gone);
   }
 
+  deleteBindingV2(req, res) {
+    const params = _(req.query)
+      .omit('plan_id', 'service_id')
+      .set('binding_id', req.params.binding_id)
+      .value();
+
+    function done() {
+      res.status(CONST.HTTP_STATUS_CODE.OK).send({});
+    }
+
+    function gone(err) {
+      /* jshint unused:false */
+      res.status(CONST.HTTP_STATUS_CODE.GONE).send({});
+    }
+    // Check if write locked
+    return lockManager.isWriteLocked(req.params.instance_id)
```

Comment:
Should we check `if (req.manager.name === CONST.INSTANCE_TYPE.DIRECTOR)`, it will save a call to apiserver.